docs/reference/cos.md

```diff
@@ -26,7 +26,11 @@ The "GitHub Self-Hosted Runner Metrics" metrics dashboard presents the following
 
 The "GitHub Self-Hosted Runner Metrics (Long-Term)" metrics dashboard displays the following rows:
 
-- General: Displays a panel showing the number of jobs per day.
+- General: Contains the following panels:
+  - Total Jobs
+  - Total unique repositories
+  - Timeseries chart displaying the number of jobs per day
+  - Percentage of jobs with low queue time (less than 60 seconds)
```

Comment:
Can you specify this in the panel title too?

Reply:
Just so I understand you correctly, it is already in the panel description, you would prefer it to be in the panel title too?

Reply:
Yes, it will be clearer to an operator what "low" means

Reply:
Have updated